README.md

```diff
@@ -3,14 +3,68 @@
 [![pytest](https://github.com/metno/py-mmd-tools/actions/workflows/tests.yml/badge.svg)](https://github.com/metno/py-mmd-tools/actions/workflows/tests.yml)
 [![codecov](https://codecov.io/gh/metno/py-mmd-tools/branch/master/graph/badge.svg)](https://codecov.io/gh/metno/py-mmd-tools)
 
-Python tools for MMD. The package contains tools for generating MMD files from netCDF-CF files with ACDD attributes, to document netCDF-CF files from MMD information, and to update the https://metno.github.io/data-management-handbook/[Data Management Handbook] with information about the translation between ACDD to MMD. Command line tools are available in the `scripts/` folder. For example,
+Python tools for MMD. The package contains tools for generating MMD files from netCDF-CF files with ACDD attributes, for documenting netCDF-CF files from MMD information, and for updating the https://metno.github.io/data-management-handbook/[Data Management Handbook] with information about the translation from ACDD to MMD. Command line tools are available in the `scripts/` folder. For example,
 
 ```
 ./script/nc2mmd.py -i tests/data/reference_nc.nc -o .
 ```
 
 generates an output MMD file called `reference_nc.xml`.
 
+# Installation
+
+To avoid problems with conflicting versions, we recommend using the [Conda](
+https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html) package manager.
+The below steps assume that Conda is installed.
+
+Create a new environment:
+
+```text
+conda create --name testenv python=3.9
+```
+
+Verify that the new environment is registered:
+
+```text
+conda info --envs
+```
+
+Activate (use) the new environment:
+
+```text
+conda activate testenv
+```
+
+Install the Py-MMD-Tools package:
+
+```text
+pip install .
+```
+
+Provided that all dependencies happen to already be installed, the following should work
```

Comment:
pip install . over vil vel gi alle dependencies som trengs for nc2mmd?

Reply:
Ja, jeg var litt usikker på det. Kanskje vi bare kan anta det? Og så heller føye til mer (i en TROBLESHOOTING-seksjon f.eks.) etter hvert som vi får erfaring fra brukere som skulle oppleve noe annet?

Reply:
:+1:

Reply:
men fikser `pip install .` metvocab også?!

Reply:
Ja, pip install . leser https://github.com/metno/py-mmd-tools/blob/master/requirements.txt

Reply:
eller kanskje https://github.com/metno/py-mmd-tools/blob/master/setup.cfg , men den e satt begge plasser